server/tests/audit_math/test_macro.py

```diff
@@ -138,6 +138,48 @@ def test_max_error(contests, batches):
             )
 
 
+def test_get_sizes_extra_contests(contests, batches):
```

Comment:
not sure i understand what this test is testing - can you explain?

Reply:
this is similar to what's going on in the `sampler` tests. There's a branch we need to test where the collected possible error is zero, so we achieve that by adding a contest that is only in some batches. If we compute the error for that contests in a batch that doesn't contain it, we get zero.